common/rpc/context_test.go

```diff
@@ -99,3 +102,19 @@ func (s *contextSuite) TestCopyContextValue_ValueNotOverWritten() {
 
 	s.Equal(newValue, newContext.Value(key))
 }
+
+func TestURL(t *testing.T) {
```

Comment:
was this supposed to be in this PR?

Reply:
I will delete this one. I was trying with url parse.